bin/server/cmd/root.go

```diff
@@ -131,6 +144,12 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&flags.demoName, "demo-name", "", "name of the demo to use when running in demo mode")
 	rootCmd.PersistentFlags().BoolVar(&flags.debug, "debug", false, "whether to enable debug level logging")
 	rootCmd.PersistentFlags().StringVar(&flags.pgURL, "pg", "", "optional external PostgreSQL instance connection url(must provide dbname); for example postgresql://user:secret@masterhost:5432/dbname?sslrootcert=cert")
+
+	// Cloud backup related flags.
+	// TODO(dragonly): Add GCS usages when it's supported.
+	rootCmd.PersistentFlags().StringVar(&flags.backupRegion, "backup-region", "", "region of the backup bucket, e.g., us-west-2 for AWS S3.")
```

Comment:
nit: put backup-region after backup-bucket

backup-region is only meaningful when backup-bucket is present. And under GCS case, people may not specify backup-region at all since GCS is global namespace.